TROUBLESHOOTING.md

```diff
@@ -0,0 +1,53 @@
+If you have a non-library SendGrid issue, please contact our [support team](https://support.sendgrid.com).
+
+If you can't find a solution below, please open an [issue](https://github.com/sendgrid/open-source-library-data-collector/issues).
+
+## Table of Contents
+
+* [Environment Variables and Your SendGrid API Key](#environment)
+* [Error Messages](#error)
+* [Version Convention](#versions)
+
+<a name="environment"></a>
+## Environment Variables and Your SendGrid API Key
+
+All of our examples assume you are using [environment variables](https://github.com/sendgrid/sendgrid-python#setup-environment-variables) to hold your SendGrid API key.
```

Comment:
For this section, can you please modify it to suggest that the user:
-  copy the .env_sample file to .env,  `cp .env_sample .env`
- open and add the params into this file
- then source the file `source ./.env`

We tend to shy away from suggesting people work around environment variables. They can figure this out on their own, but we would much rather push for .env usage.